sockets.js

```diff
@@ -212,7 +216,41 @@ if (cluster.isMaster) {
 	}
 
 	let app = require('http').createServer();
-	let appssl = Config.ssl ? require('https').createServer(Config.ssl.options) : null;
+	let appssl = null;
+	if (Config.ssl) {
+		let key;
+		if (require('path').isAbssolute(Config.ssl.options.key)) {
```

Comment:
Why does this need to be absolute, anyway? You can just run `path.resolve` on relative paths before passing them.

Reply:
I think it was because I thought they had to be kept in a specific path or something? But `path.resolve` wouldn't be a problem with that anyways. Trying to code yesterday wasn't one of my best ideas